src/app.module.ts

```diff
@@ -22,6 +22,9 @@ import { EventStreamModule } from './event-stream/event-stream.module';
 import { EventStreamProxyModule } from './eventstream-proxy/eventstream-proxy.module';
 import { HealthModule } from './health/health.module';
 import { HealthController } from './health/health.controller';
+import { APP_INTERCEPTOR } from '@nestjs/core';
+import { PrometheusModule } from '@willsoto/nestjs-prometheus';
+import { LoggingAndMetricsInterceptor, MetricProviders } from './logging-and-metrics.interceptor';
```

Comment:
Can you run `npm run lint:fix` (I think it will sort imports)

Reply:
Sure - done :)